File: js/removeCharacters.js

var fs = require('fs');
var output = [];

function removeCharacters( sentence, chars ) {
	'use strict';
	var regex = new RegExp('['+chars+']+', 'g');
	return sentence.replace(regex,'');
}


fs.readFileSync( process.argv[2] )
	.toString()
	.split( '\n' )
	.forEach( function( line ) {
		if ( line === '' ) return;

		output.push( removeCharacters.apply(null, line.split(', ') ) );
	});

console.log( output.join( '\n' ) );

File: python/removeCharacters.py
import sys
import re

def removeCharacters(sentence, chars):
	regex = re.compile('['+chars+']')
	return regex.sub('', sentence)
	
# Open file for reading.
test_cases = open(sys.argv[1], 'r')

for line in test_cases:
	args = line.split(', ')
	print(removeCharacters(args[0], args[1]))